docs/diagrams/FindSequenceDiagram.puml

```diff
@@ -0,0 +1,79 @@
+@startuml
+'https://plantuml.com/sequence-diagram
+!include style.puml
+skinparam ArrowFontStyle plain
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant ":FindCommandParser" as FindCommandParser LOGIC_COLOR
+participant "f:FindCommand" as FindCommand LOGIC_COLOR
+participant "r:CommandResult" as CommandResult LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":PersonContainsKeywordPredicate" as PersonContainsKeywordPredicate MODEL_COLOR
+participant "m:Model" as Model MODEL_COLOR
+end box
+
+[-> LogicManager : execute("find n/John Doe")
+activate LogicManager
+
+LogicManager -> AddressBookParser : parseCommand("find n/John Doe")
```

Comment:
I like the name that you chose to find, given you are singaporean and no one probably has that name